Packages.targets

```diff
@@ -1,29 +1,29 @@
 <Project>
     <ItemGroup>
-        <PackageReference Update="FSharp.Core" Version="4.6.2" />
-        <PackageReference Update="FSharp.Data" Version="3.0.0" />
-        <PackageReference Update="Xamarin.Forms" Version="4.3.0.908675" />
-        <PackageReference Update="Xamarin.Forms.Platform.WPF" Version="4.3.0.908675" />
-        <PackageReference Update="Xamarin.Forms.Maps" Version="4.3.0.908675" />
+        <PackageReference Update="FSharp.Core" Version="4.7.0" />
+        <PackageReference Update="FSharp.Data" Version="3.3.2" />
+        <PackageReference Update="Xamarin.Forms" Version="4.3.0.991211" />
```

Comment:
Those versions are used by NuGet (when packaging) to determine the minimum dependency requirement.

So it is better to target the lowest major version (e.g. for XF = 4.3), otherwise, people won't be able to target a version lower than 4.3.0.991211 when using Fabulous.XamarinForms (for example, if a critical bug has been introduced in this version)

Maybe using wildcards would be a better solution?
`<PackageReference Update="Xamarin.Forms" Version="4.3.*" />`

The NuGet packages would only require >= 4.3.0 but the samples would be able to use the latest versions available.

This is applicable to all "users"-facing dependencies like FSharp.Core/Data, SkiaSharp, etc.
But not to things like Mono.Cecil, SourceLink, FsUnit, etc. which can explicitly target the latest versions.

Reply:
ok yes, thanks, will do

ps. I think FSharp.Data is not (or should not) be public facing for Fabulous?

Reply:
Yes, my bad. It should only be used by samples.